lib/crystalball/predictor/direct_map.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Crystalball
+  class Predictor
+    # Used with `predictor.use Crystalball::Predictor::DirectMap.new(from: %r{models/(.*).rb}, to: "./spec/models/%s_spec.rb")`.
+    # When used will look for files matched to `from` regex and use captures to fill `to` string to
+    # get paths of proper specs
+    class DirectMapPredictor
```

Comment:
its not a predictor. It's a prediction strategy. `Crystalball::Predictor` is a predictor. Please rename a class to something like `AssociatedSpecs`

Reply:
And to use it with the same interface as `Predictor`. Hmm. Maybe we need to rename it
I don't like an idea to have different conceptions to do the similar staff

Reply:
done